content/posts/2021/10/libplanet-0.11-0.19/index.eng.md

```diff
@@ -0,0 +1,251 @@
+in Planetarium work closely together to catch up each other's updates.  This
+release cycle of Libplanet is likely to continue for a while.
+
+Under that policy, there have been 9 minor releases in the last few months,
+from 0.11.0 released in March 2021 to 0.19.0 released this week:
+
+- [0.11.0] (March 30)
+- [0.12.0] (July 23)
+- [0.13.0] (July 28)
+- [0.14.0] (August 5)
+- [0.15.0] (August 18)
+- [0.16.0] (August 25)
+- [0.17.0] (September 28)
+- [0.18.0] (October 13)
+- [0.19.0] (October 27)
+
+[0.11.0]: https://github.com/planetarium/libplanet/releases/tag/0.11.0
+[0.12.0]: https://github.com/planetarium/libplanet/releases/tag/0.12.0
+[0.13.0]: https://github.com/planetarium/libplanet/releases/tag/0.13.0
+[0.14.0]: https://github.com/planetarium/libplanet/releases/tag/0.14.0
+[0.15.0]: https://github.com/planetarium/libplanet/releases/tag/0.15.0
+[0.16.0]: https://github.com/planetarium/libplanet/releases/tag/0.16.0
+[0.17.0]: https://github.com/planetarium/libplanet/releases/tag/0.17.0
+[0.18.0]: https://github.com/planetarium/libplanet/releases/tag/0.18.0
+[0.19.0]: https://github.com/planetarium/libplanet/releases/tag/0.19.0
+
+
+Protocol Versions
+-----------------
+
+*Added in version [0.11.0].*
+
+A protocol version is a property of a block that represents in which version of
+Libplanet used for mining.  It helps games powered by Libplanet to easily
+upgrade the Libplanet version, and on the Libplanet side, it allows us to
+add new functionalities without breaking backward compatibility with existing
+networks.
+
+API-wise, it is exposed through
+the [`Block<T>.ProtocolVersion` property][Block<T>.ProtocolVersion], which is
+a 32-bit integer.  Those numbers solely represent a version of the network
+protocol, which is distinct from a version of the library package (on NuGet
+Gallery).  According to Semantic Versioning, patch releases never increase
+the protocol version, but only minor and major release can increase it.[^2]
+
+As of Libplanet [0.19.0], the current protocol version is 2, and any blocks
+produced before Libplanet [0.11.0], which introduced protocol versions,
+are treated as the protocol version 0.
+
+[^2]: This does not mean that all major and minor releases increase the protocol
+      version. In fact, most releases do not increment the protocol version.
+
+[Block<T>.ProtocolVersion]: https://docs.libplanet.io/0.19.0/api/Libplanet.Blocks.Block-1.html#Libplanet_Blocks_Block_1_ProtocolVersion
+
+
+Network Transport Layer
+-----------------------
+
+*Added in version [0.11.0].*
+
+Since its inception, Libplanet has left much of the low-level networking to
+[NetMQ], which is a .NET implementation of [ZeroMQ].  However, now it is
+necessary to change its networking layer to be possible without relying on NetMQ
+so that Libplanet can be used on various platforms such as mobile and Web.
+
+The network transport layer is Libplanet's response to that need.
```

Comment:
```suggestion
The network transport layer allows using any low-level networking implementation.
```

I felt the word 'that need' bit awkward and thought it would be better to match the Korean source. Or we could say something like:

```suggestion
We created the network transport layer to meet those demands.
```